classes/class-nets-easy-api.php

```diff
@@ -248,6 +246,23 @@ public function get_nets_easy_unscheduled_subscription_by_external_reference( $d
 		return $this->check_for_api_error( $response );
 	}
 
+	/**
+	 * Terminates Nexi session.
+	 *
+	 * @param string $payment_id The payment identifier.
+	 *
+	 * @return array|mixed
+	 */
+	public function terminate_nets_easy_session( $payment_id ) {
+		$request  = new Nets_Easy_Request_Terminate_Session(
+			array(
+				'payment_id' => $payment_id,
+			)
+		);
+		$response = $request->request();
+		return $this->check_for_api_error( $response );
```

Comment:
Suggestion: If something goes wrong during the checkout with this request, this will trigger a error message in the frontend as it is right now. Since we just want to do this in the background and there really isn't an action a customer can take when this happens to solve it i would suggesting to just return the response directly instead.